styles/styles.css

```diff
@@ -795,6 +795,41 @@ main .section.responsive-title h1 {
 }
 
 /* REDESIGN GLOBAL STYLES */
-:root.redesign-v2 {
+.redesign-v2 {
+  --v2-space-small: 16px;
```

Comment:
Nice to have this custom property. Could you add that for other spacings as well and use it in the block CSS?

Reply:
Well I don't know what could be the other values I just add the `16px` because it looks like the smallest padding but other sizes should be discussed.